client/config.c

```diff
@@ -91,6 +139,11 @@ TDNFReadConfig(
         }
     }
 
+    dwError = TDNFParseOSInfo(pConf);
+    if (dwError != ENOENT) {
```

Comment:
I have a question. If we the os-release file doesn't exist and it's considered not a fatal error, why should we consider other errors in `TDNFParseOSInfo` as fatal? why not use some default values in case of failure and make the return type void?

Reply:
I think `/etc/os-release` not present is an acceptable condition, but failing to parse it for any reason is not. It may be some deeper issue (a bug in the parser, file corrupted or whatever) , and we shouldn't proceed.